default.nix

```diff
@@ -34,13 +34,18 @@ let
             (with nixpkgs.haskell.lib; self: super: {
                   streamly-coreutils =
                       utils.local super "streamly-coreutils" ./. "--benchmark" inShell;
-                  streamly = utils.composewell super
-                      "streamly"
-                      "9596aea3ab33d3c2fa513247fcfe8604bd7ff454";
-                  streamly-core = utils.composewellDir super
-                      "streamly"
-                      "9596aea3ab33d3c2fa513247fcfe8604bd7ff454"
-                      "/core";
+                  streamly-core =
+                      super.callHackageDirect
+                        { pkg = "streamly-core";
+                          ver = "0.1.0";
+                          sha256 = "hoSV6Q2+X5a7hFnJAArqNPjcMaCVyX9Vz4FcxeJ+jgI=";
+                        } {};
```

Comment:
You can keep the previous code as commented here. because sooner or later we will have to add it as github commit again.